src/app/pages/ArticlePage/fixtureData.ts

```diff
@@ -46,6 +46,7 @@ const articleDataBuilder = (
   promoHeadline: string,
   summary: string,
   things: object,
+  readTime: number,
```

Comment:
Will this fix the undefined issue - if you don't pass anything in?
```suggestion
  readTime?: number,
```

Reply:
unfortunately not
<img width="639" height="44" alt="Screenshot 2025-07-23 at 4 31 39 pm" src="https://github.com/user-attachments/assets/9d654f13-dc02-4514-ac7a-fea42fc6b10c" />

if there are any arguments supplied after `readTime` then `articleDataBuilder` thinks that your still trying to pass in `readTime` and will throw an error

Reply:
Try the default to 0 scenario - but in reality, pretty much all optimo articles should have a readTime in the metadata, right?

Reply:
Also move readTime down to the bottom, next to `type` - that way the "Default parameters should be last" error won't occur

Reply:
from previous discussions about this ticket, I remember it being mentioned that `readTime` is a relatively new bit of metadata so there should be a fair amount of articles made before `readTime` was added?

Reply:
Ah yes, that might well be the case for some older CPS content (although I checked the data for https://www.bbc.com/mundo/noticias-56669604 and it was there). Anyways in the case of this - fixture data - we could assume that readTime is always there (because for new articles, it will be) - the default could be 1 or 2, and then the exception should be the absence of readTime - but I think we would just override this in the unit tests rather than try to cater for it here in the articleBuilder?

Reply:
So, with this in mind - can we just remove read time as a parameter, and hard code it. 

And if we want to test a scenario without it, we can just remove it explictly in the test

Reply:
Let me know if you want to discuss - might be easier than lots of comments back and forth!

Reply:
okay, I'm good to go with this